test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_notification.feature

```diff
@@ -394,13 +464,126 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
+      | house=flat  |
+      | house"flat" |
+      | house'flat' |
+      | house;flat  |
+      | house(flat) |
 
- #  --- (Pending to changes and/ or development) still missing tests for:
- #  - notification field and its subfields
- #  - expires field
- #  - status field
+  @notification_attrs_not_plain_ascii @BUG_2099 @skip
+  Scenario Outline:  try to create a new subscription using NGSI v2 with not plain ascii values in notification attrs field
+    Given  a definition of headers
+      | parameter          | value                         |
+      | Fiware-Service     | test_notification_attrs_error |
+      | Fiware-ServicePath | /test                         |
+      | Content-Type       | application/json              |
+    # These properties below are used in subscriptions request
+    And properties to subscriptions
+      | parameter             | value                   |
+      | subject_idPattern     | .*                      |
+      | condition_attrs       | temperature             |
+      | notification_http_url | http://localhost:1234   |
+      | notification_attrs    | <attribute>             |
+      | expires               | 2016-04-05T14:00:00.00Z |
+    When create a new subscription
+    Then verify that receive a "Bad Request" http code
+    And verify an error response
+      | parameter   | value                                        |
+      | error       | BadRequest                                   |
+      | description | Invalid value parsing notification attribute |
+    Examples:
+      | attribute  |
+      | habitación |
+      | españa     |
+      | barça      |
 
+  @notification_attrs_length_exceed @BUG_2100 @skip
+  Scenario:  try to create a new subscription using NGSI v2 with length max exceed (256) in notification attrs field
+    Given  a definition of headers
+      | parameter          | value                         |
+      | Fiware-Service     | test_notification_attrs_error |
+      | Fiware-ServicePath | /test                         |
+      | Content-Type       | application/json              |
+    # These properties below are used in subscriptions request
+    And properties to subscriptions
+      | parameter             | value                   |
+      | subject_idPattern     | .*                      |
+      | condition_attrs       | temperature             |
+      | notification_http_url | http://localhost:1234   |
+      | notification_attrs    | random=257              |
+      | expires               | 2016-04-05T14:00:00.00Z |
+    When create a new subscription
+    Then verify that receive a "Bad Request" http code
+    And verify an error response
+      | parameter   | value                                                              |
+      | error       | BadRequest                                                         |
+      | description | notification attribute name length: 257, max length supported: 256 |
 
+  @notification_attrs_two_equals @BUG_2101 @skip
```

Comment:
I believe we decided this is the expected behavior of the broker.
If you ask to be notified with an attribute 'twice', the broker just does what you asked it to do.
The issue #2101 should be closed (check with @Fermin first).